internal/exec/util/passwd.go

```diff
@@ -50,13 +50,24 @@ func appendIfStringSet(args []string, arg string, str *string) []string {
 }
 
 // EnsureUser ensures that the user exists as described. If the user does not
-// yet exist, they will be created, otherwise the existing user will be
-// modified.
+// yet exist, they will be created, otherwise the existing user will be modified.
+// If shouldExist variable is set to false and the user already exists, then they
+// will be deleted.
 func (u Util) EnsureUser(c types.PasswdUser) error {
```

Comment:
Hmm, should this be renamed e.g. `ProcessUser` or something? Since it doesn't just create users anymore.

Reply:
My thinking on the naming was that we are ensuring that the user is in the specified state.